src/elements/content-sidebar/__tests__/BoxAISidebar.test.tsx

```diff
@@ -5,7 +5,9 @@ import BoxAISidebarComponent, { BoxAISidebarProps } from '../BoxAISidebar';
 describe('elements/content-sidebar/BoxAISidebar', () => {
     const renderComponent = (props = {}) => {
         const defaultProps = {
+            onBoxAISidebarOpened: jest.fn(),
```

Comment:
Should we make this more generalized (i.e. `onOpen`) by adding it as a callback prop to ContentSidebar if it doesn't already exist?

Reply:
We want to do it explicitly for opening Box AI Sidebar using `shouldOpenBoxAISidebar`, however I can rename it to `onOpen` since it's already in `boxAISidebarProps`